R/analyse.R

```diff
@@ -459,11 +459,17 @@ validate_analyse_pars <- function(results, pars) {
     )
 
     for (par in pars) {
-        if (par != "df") {
+        if (!par %in% c("df", "se")) {
             assert_that(
                 all(!is.na(vapply(results_unnested, function(x) x[[par]], numeric(1)))),
                 msg = sprintf("Parameter `%s` contains missing values", par)
             )
+        } else if(par == "se") {
+            assert_that(
+                all(!is.na(vapply(results_unnested, function(x) x[[par]], numeric(1)))) ||
+                all(is.na(vapply(results_unnested, function(x) x[[par]], numeric(1)))) ,
+                msg = sprintf("Parameter `%s` contains both missing and observed values", par)
+            )
```

Comment:
Should the df also be handled by the second else if condition ? As in we allow DF to be NA but surely we don't allow for a mix of  missing and non-missing ?

Reply:
Agreed!